src/torchcodec/encoders/_video_encoder.py

```diff
@@ -16,9 +16,18 @@ class VideoEncoder:
             C is 3 channels (RGB), H is height, and W is width.
             Values must be uint8 in the range ``[0, 255]``.
         frame_rate (float): The frame rate of the **input** ``frames``. Also defines the encoded **output** frame rate.
+        device (str or torch.device, optional): The device to use for encoding. Default: "cpu".
+            If you pass a CUDA device, frames will be encoded on GPU.
+            Note: The "beta" CUDA backend is not supported for encoding.
```

Comment:
There's no need to mention that, the `set_cuda_backend` selector is currently only defined in `torchcodec.decoders` so it's only clearly about decoding.